src/cryptography/hazmat/primitives/padding.py

```diff
@@ -122,3 +126,53 @@ def finalize(self):
         res = self._buffer[:-pad_size]
         self._buffer = None
         return res
+
+
+class PKCS7(_BytePadding):
```

Comment:
But I can find existing exception to this rule like _AEADEncryptionContext
If I can not subclass in this case I will have to duplicate a good 200 lines of code (or design a poor manual subclassing behaviour), I think this is much bigger problem than your rule. I understand they are rules but I guess they should be considered as best practice and not fundamental at the point to not be broken when it is obviously better.